proto/metalstack/api/v2/project.proto

```diff
@@ -38,6 +38,11 @@ service ProjectService {
     option (project_roles) = PROJECT_ROLE_EDITOR;
     option (auditing) = AUDITING_INCLUDED;
   }
+  // Leave project
+  rpc LeaveProject(ProjectServiceLeaveProjectRequest) returns (ProjectServiceLeaveProjectResponse) {
```

Comment:
Shall we rename this to a simple `Leave`? The `Project` feels repetitive. What are your opinions?